setup.py

```diff
@@ -1,21 +1,37 @@
+import re
 from distutils.core import Extension, setup
 from pathlib import Path
 
+# from https://stackoverflow.com/a/7071358/2750945
+VERSIONFILE = "pyrp3/_version.py"
+verstrline = open(VERSIONFILE, "rt").read()
+VSRE = r"^__version__ = ['\"]([^'\"]*)['\"]"
+mo = re.search(VSRE, verstrline, re.M)
+if mo:
+    verstr = mo.group(1)
+else:
+    raise RuntimeError("Unable to find version string in %s." % (VERSIONFILE,))
 this_directory = Path(__file__).parent
 long_description = (this_directory / "README.rst").read_text()
 
 setup(
     name="pyrp3",
-    version="1.1.1",
+    version="1.2.0",
     description="Python utilities for redpitaya",
     author="Pierre Cladé",
     author_email="pierre.clade@upmc.fr",
     maintainer="Bastian Leykauf",
     maintainer_email="leykauf@physik.hu-berlin.de",
     long_description=long_description,
     long_description_content_type="text/x-rst",
-    packages=["pyrp3", "pyrp3.enum"],
-    install_requires=["myhdl", "rpyc", "cached_property", "numpy"],
+    packages=["pyrp3"],
+    python_requires=">=3.5",
+    install_requires=[
+        "myhdl>=0.11",
+        "rpyc>=4.0,<5.0",
```

Comment:
I thought that rpyc >5.0 compatibility was fixed in version 0.3.2 of linien (according to [linien's CHANGELOG.md](https://github.com/linien-org/linien/commit/f530ea13da636b155a17219aa93a1e0c0c62a8ae)).

Reply:
The problem with rpyc>=5.0 was that it was incompatible with the outdated python version of RedPitaya OS 1.0. The fix in that version is that the installed version was explicitly set to <5.0 to avoid conflicts between server and client. RedPitaya OS 2.0 apparently now has a stable release and the next version of Linien will only support OS 2.0.

That means that the requirement will be `rpyc>=5.0,<6.0` from then on.

Reply:
> The problem with rpyc>=5.0 was that it was incompatible with the outdated python version of RedPitaya OS 1.0

Python supports constraining that with the python version, as explained here: https://stackoverflow.com/questions/21082091/install-requires-based-on-python-version

Reply:
But we have to make sure that we always use the same version of rpyc for server *and* client since they are incompatible. Will be upgraded to >=5.0,<6.0 with the next release.